File: 프로그래머스/연습문제/Lv1/2016년/2016년.cpp
#include <string>
#include <vector>

using namespace std;

string solution(int a, int b) {
    int days[11] = {31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30};
    string answer[7] = {"FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"};
    
    int cnt = 0;
    for(int i=0; i<a-1; cnt+=days[i++]);
    cnt = cnt+b-1;
    
    return answer[cnt%7];
}

File: 프로그래머스/연습문제/Lv1/2016년/2016년.py
def solution(a, b):
    days = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30]
    answer = ["FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"]

    cnt = 0
    for i in range(a-1):
        cnt += days[i]
    cnt = cnt+b-1

    return answer[cnt%7]
